texthero/preprocessing.py

```diff
@@ -11,14 +11,44 @@
 import numpy as np
 import pandas as pd
 import unidecode
+
+# Ignore gensim annoying warnings
+import warnings
 from nltk.stem import PorterStemmer, SnowballStemmer
 
 from texthero import stopwords as _stopwords
 
 from typing import List, Callable
 
-# Ignore gensim annoying warnings
-import warnings
+"""
+Define all regex pattern, which will be used in the functions below. They define different charateristics, on how to clean
```

Comment:
We should limit the docstring to 75 as per the CONTRIBUTING guidelines (Cedric is working on that #142 )

Reply:
I limited the line lenght of my docstring. I hope that is what you meant 🙊